l3kernel/l3text.dtx

```diff
@@ -342,13 +342,54 @@
 %   See also \cs{text_map_function:nN}.
 % \end{function}
 %
+% Word breaking is implemented using the standard algorithm described in Unicode
+% Standard Annex \#29. Text starting with a line feed or carriage return
+% character will drop this due to standard \TeX{} processing. Spaces are always
+% considered a break even if immediately followed by an extending character.
+% At present extended pictograms are not supported: these may be added in a future
+% release. Language-based tailoring may be added in a future release: at present
+% the algorithm is exactly that described in the annex.
+%
+% \begin{function}[rEXP, added = 2025-02-12]{\text_words_map_function:nN}
+%   \begin{syntax}
+%     \cs{text_words_map_function:nN} \Arg{text} \meta{function}
+%   \end{syntax}
+%   Takes user input \meta{text} and expands as described for
+%   \cs{text_expand:n}, then maps over the \emph{words} within the
```

Comment:
Better:
This takes the user input in . . . 
expands it as with . . . ; it then maps over . . .